CMakeLists.txt

```diff
@@ -53,9 +60,15 @@ endif()
 target_link_libraries(stem
   PRIVATE ${_libs})
 
-add_executable(stemcli
-  stempy/main.cpp
-)
+if(TARGET vtkm::cuda)
```

Comment:
I don't think that the main needs to be compiled with cuda. It is fine to have this as just cxx